tests/drivers/gpio/gpio_basic_api/src/test_gpio_port.c

```diff
@@ -81,8 +81,11 @@ static int setup(void)
 	zassert_equal(rc, 0,
 		      "get raw low failed");
 	if (raw_in() != false) {
-		printk("FATAL: output low does not read low\n");
-		k_panic();
+		TC_PRINT("FATAL no short: output low does not read low\n"
```

Comment:
I'm not sure "no short" will be clear enough for non native english speakers. At least to me it is not clear.
While we should not miss an occasion to  progress in a foreign language, I wonder if a more 'common' expression could be used here.

Reply:
maybe something like:
`FATAL input pin not wired to output pin: ...`

Reply:
Updated:
```
starting test - test_gpio_port
Validate device GPIO_2
Check GPIO_2 output 16 connected to input 17
FATAL output pin not wired to input pin? (out low => in high)
```

Reply:
Perfect, thanks!